google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/stub/GrpcDlpServiceStub.java

```diff
@@ -0,0 +1,241 @@
+                      ListInspectFindingsRequest.getDefaultInstance()),
+                  io.grpc.protobuf.ProtoUtils.marshaller(
+                      ListInspectFindingsResponse.getDefaultInstance())));
+  private static final UnaryCallable<ListInfoTypesRequest, ListInfoTypesResponse>
+      directListInfoTypesCallable =
+          GrpcCallableFactory.createDirectCallable(
+              io.grpc.MethodDescriptor.create(
+                  io.grpc.MethodDescriptor.MethodType.UNARY,
+                  "google.privacy.dlp.v2beta1.DlpService/ListInfoTypes",
+                  io.grpc.protobuf.ProtoUtils.marshaller(ListInfoTypesRequest.getDefaultInstance()),
+                  io.grpc.protobuf.ProtoUtils.marshaller(
+                      ListInfoTypesResponse.getDefaultInstance())));
+  private static final UnaryCallable<ListRootCategoriesRequest, ListRootCategoriesResponse>
+      directListRootCategoriesCallable =
+          GrpcCallableFactory.createDirectCallable(
+              io.grpc.MethodDescriptor.create(
+                  io.grpc.MethodDescriptor.MethodType.UNARY,
+                  "google.privacy.dlp.v2beta1.DlpService/ListRootCategories",
+                  io.grpc.protobuf.ProtoUtils.marshaller(
+                      ListRootCategoriesRequest.getDefaultInstance()),
+                  io.grpc.protobuf.ProtoUtils.marshaller(
+                      ListRootCategoriesResponse.getDefaultInstance())));
+  private final BackgroundResource backgroundResources;
+  private final GrpcOperationsStub operationsStub;
+
+  private final UnaryCallable<InspectContentRequest, InspectContentResponse> inspectContentCallable;
+  private final UnaryCallable<RedactContentRequest, RedactContentResponse> redactContentCallable;
+  private final UnaryCallable<CreateInspectOperationRequest, Operation>
+      createInspectOperationCallable;
+  private final OperationCallable<
+          CreateInspectOperationRequest, InspectOperationResult, InspectOperationMetadata,
+          Operation>
+      createInspectOperationOperationCallable;
+  private final UnaryCallable<ListInspectFindingsRequest, ListInspectFindingsResponse>
+      listInspectFindingsCallable;
+  private final UnaryCallable<ListInfoTypesRequest, ListInfoTypesResponse> listInfoTypesCallable;
+  private final UnaryCallable<ListRootCategoriesRequest, ListRootCategoriesResponse>
+      listRootCategoriesCallable;
+
+  public static final GrpcDlpServiceStub create(DlpServiceSettings settings) throws IOException {
+    return new GrpcDlpServiceStub(settings, ClientContext.create(settings));
+  }
+
+  public static final GrpcDlpServiceStub create(ClientContext clientContext) throws IOException {
+    return new GrpcDlpServiceStub(DlpServiceSettings.defaultBuilder().build(), clientContext);
+  }
+
+  /**
+   * Constructs an instance of GrpcDlpServiceStub, using the given settings. This is protected so
+   * that it easy to make a subclass, but otherwise, the static factory methods should be preferred.
```

Comment:
This is a constructor so there's no overriding. However, if the constructor is private it isn't possible to create a subclass. I can imagine something like

```
Subclass() {
  super(...);
  someOtherField = 42;
}
```